src/main/kotlin/no/nav/familie/ks/sak/api/dto/EndretUtbetalingAndelRequestDto.kt

```diff
@@ -26,10 +30,15 @@ fun EndretUtbetalingAndelRequestDto.mapTilBegrunnelser(): List<NasjonalEllerFell
     return when (this.årsak) {
         Årsak.DELT_BOSTED,
         Årsak.ENDRE_MOTTAKER,
-        Årsak.ALLEREDE_UTBETALT,
         Årsak.ETTERBETALING_3MND,
         -> listOf(NasjonalEllerFellesBegrunnelse.AVSLAG_SØKT_FOR_SENT_ENDRINGSPERIODE)
 
+        Årsak.ALLEREDE_UTBETALT,
+        ->
+            this.begrunnelser.ifEmpty {
+                listOf(NasjonalEllerFellesBegrunnelse.AVSLAG_SØKT_FOR_SENT_ENDRINGSPERIODE)
+            }
```

Comment:
Jeg tenker her at man ikke trenger å defaulte til NasjonalEllerFellesBegrunnelse.AVSLAG_SØKT_FOR_SENT_ENDRINGSPERIODE. 
```suggestion
            this.begrunnelser
```

Begrunnelsen er uansett ikke gyldig for Årsak.ALLEREDE_UTBETALT, men bare gyldig for Årsak.ETTERBETALING_3MND (som allerede settes til den begrunnelsen).

Reply:
Bra at du adresserte dette, jeg var usikker selv men lot det være en default siden det sto der fra før av. Jeg endrer 🙌